commit 7fefcc028010328422766b551541231309e2158b
Author: John Greb <h3x4m3r0n@gmail.com>
Date:   Wed Mar 8 15:12:29 2017 +0000

    Match dwc2 device-tree fifo sizes to the hardware values.
    
    Since commit aa381a7259c3f53727bcaa8c5f9359e940a0e3fd was reverted with 3fa9538539ac737096c81f3315a14670b1609092 the g-tx-fifo-size array in the device-tree needs to match the preset values in the bcm2835.
    
    Resolves https://github.com/raspberrypi/linux/issues/1876

diff --git a/arch/arm/boot/dts/overlays/dwc2-overlay.dts b/arch/arm/boot/dts/overlays/dwc2-overlay.dts
index 527abc9f0ddf..265a16bab008 100644
--- a/arch/arm/boot/dts/overlays/dwc2-overlay.dts
+++ b/arch/arm/boot/dts/overlays/dwc2-overlay.dts
@@ -15,7 +15,7 @@
 			dr_mode = "otg";
 			g-np-tx-fifo-size = <32>;
 			g-rx-fifo-size = <256>;
-			g-tx-fifo-size = <256 128 128 64 64 64 32>;
+			g-tx-fifo-size = <512 512 512 512 512 768>;
 			status = "okay";
 		};
 	};
@@ -24,6 +24,5 @@
 		dr_mode = <&dwc2_usb>, "dr_mode";
 		g-np-tx-fifo-size = <&dwc2_usb>,"g-np-tx-fifo-size:0";
 		g-rx-fifo-size = <&dwc2_usb>,"g-rx-fifo-size:0";
-		g-tx-fifo-size = <&dwc2_usb>,"g-tx-fifo-size:0";
 	};
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1130_7fefcc028010328422766b551541231309e2158b.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1130_7fefcc028010328422766b551541231309e2158b.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
